src/http/http_endpoint.h

```diff
@@ -249,7 +249,10 @@ namespace http
         }
 
         const auto& actor_s = actor_opt.value();
-        auto actor = rpc_map->resolve(actor_s);
+        LOG_DEBUG_FMT("Processing {}", actor_s);
```

Comment:
Is this too verbose?

Reply:
Absolutely! These were just for debugging, removing now.